### PR TITLE
Changes

### PR DESCRIPTION
**Pull request by marcojonudo**
Be fast, my friend! @casey-collab

### CODE REVIEW

File: espanol/pasadizo/ruta/caida/aires.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+El Consejero vol� todos los metros que le permit�a su masa corporal, estrey�ndose
```
Comment (casey-collab) [inline espanol/pasadizo/ruta/caida/aires.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: espanol/pasadizo/ruta/caida/escalera.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+El Consejero rod� escaleras abajo, sufriendo el impacto contra los m�ltiples escalones.
```
Comment (casey-collab) [inline espanol/pasadizo/ruta/caida/escalera.md]: This file has also been added to a directory that already contained a page.

File: espanol/pasadizo/ruta/plaza.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+El Consejero y su corte cruzaron corriendo la plaza, esperando confundir al enemigo
```
Comment (casey-collab) [inline espanol/pasadizo/ruta/plaza.md]: This file has also been added to a directory that already contained a page.

File: espanol/pasadizo/ruta/sotano.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+En el s�tano tiraron la estanter�a, aquella que tapaba la puerta al pasadizo. El
```
Comment (casey-collab) [inline espanol/pasadizo/ruta/sotano.md]: This file has also been added to a directory that already contained a page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by marcojonudo** [comment]
Take another look, @casey-collab

File: espanol/pasadizo/plaza/escaleras/escalera.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+El Consejero rod� escaleras abajo, sufriendo el impacto contra los m�ltiples escalones.
```
Comment (casey-collab) [inline espanol/pasadizo/plaza/escaleras/escalera.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by marcojonudo** [comment]
This is it, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :ice_cream:

**Pull request merged**